CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 1.2.0
```

Comment:
Mention that now the SDK works also to annotate AutoTrace spans in combination with the Java Tracer version X.Y.Z and above. We know that it is not related to the SDK, but this is not something we should assume our customers to understand ;-)

Reply:
Good point, added.